### PR TITLE
Add package entry for SCIM pickens

### PR DESCRIPTION
**Pull request by web-kat**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,3 +1,5 @@
 * @powerhome/heroes-for-hire
 packages/**/*.gemspec
 packages/**/Gemfile
+
+packages/scim_pickens/ @powerhome/s-h-i-e-l-d
```
Comment (web-kat) [inline .github/CODEOWNERS:5]: @benlangfeld @wadewinningham is it possible to get the shield team added to this repository?

Comment (benlangfeld) [reply .github/CODEOWNERS:5]: I don't think it is necessarily appropriate to start allocating shared ownership of power-tools.

What are you trying to achieve with this library?

File: packages/scim_pickens/docs/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# SCIM Pickens 🤠
+
+SCIM Pickens is a [SCIM](https://www.simplecloud.info/) library. TODO: fill this out with a better description
```
Comment (benlangfeld) [inline packages/scim_pickens/docs/README.md:3]: A client library for use in Nitro? Should that really be part of the generic power-tools, and not part of NitroID's suite of client code?

**Comment by web-kat** [comment]
Moving into nitro-id. Closing.

**Pull request closed without merging**